### DISCUSSION TITLE
blog/black-myth-wukong-review/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#357] [Announcements]
# blog/black-myth-wukong-review/

《黑神话：悟空》一周目体验总结：虽然游戏有不少亮点，但整体内容稍显不足，导致游戏进程偏快，令人意犹未尽。

https://jimmysong.io/blog/black-myth-wukong-review/

**Comment by LiZhenCheng9527** [comment]
有恢复法力的酒酿，叫琼浆。战斗系统其实鼓励玩家根据对应的boss采取不同的棍法，或者说技能加点。这游戏的所有成长要素都能够无消耗的全部重置。而且不同的boss招数特点都很鲜明。我在战斗中遇到卡关的情况就会去思考使用什么棍法能够更好识破boss的招式。就像有长连段但追击能力不强的虎先锋，戳棍的无消耗寸退就比劈棍切手格挡更加的安全和操作简单。我觉得法宝系统也优点可惜，做的没有太多的深度。只有四个，提供一些功能性。希望在DLC中能够有相关系统的深化。

**Reply by rootsongjc** [reply]
嗯，我是在一周目快结束的时候才发现琼浆。不过因为我一直用的劈棍，所以对其他棍法的使用也不是很了解。期待DLC早日推出的，据说会在农历新年前后推出

**Reply by LiZhenCheng9527** [reply]
有爆料说是明年2月份，赶春节。那可能就是之前删减掉的部分再利用。期待能够真正讲好讲完一个故事。

**Reply by rootsongjc** [reply]
但愿吧，玩完黑神话后我的 PS5 都已经吃灰好久了。

**Comment by liks144** [comment]
作为一款ARPG来说内容已经非常丰富了, 小怪90种头目81种, 所有小怪都有完整的动作模型, 头目也有符合自己设定的招式, 一周目打得我眼花缭乱, 比起某些知名的日系RPG 两百块的售价很有诚意了... 您想说的可能是剧情内容吧? 没有特别长的支线剧情, 基本上就是打打打 时间久了可能就觉得无聊, 而且后面火焰山建模也有点奇怪, 和火焰山当地的建筑差别比较大, 合理怀疑有点赶进度... 最喜欢的还是黄风岭, 故事, 环境设计. 地图和导航很影响沉浸体验, 而且也不是开放世界, 没有也正常

**Reply by liks144** [reply]
对 后面浮屠界和盘丝洞的设计感觉没前面好玩

**Reply by rootsongjc** [reply]
对，我的意思就是剧情太短了，在小西天那章感觉有点故弄玄虚，也可能是因为我没看懂吧。不过动画做的不错。
